startup.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+
+from discord.ext import commands
+
+from cogs.utils.config import *
+
+
+# First time run config
+def wizard():
+    config = {}
+    print('Welcome to AlanAyy\'s Ignite-Ember setup!')
```

Comment:
Would you be ok taking your name off the print? Preferably this belongs to Schulich Ignite. Your name defs belongs on a `contributors.md` file tho :)
```suggestion
    print('Welcome to Ignite-Ember setup!')
```